Scripts/s4ap/events/items/receive_item_event.py

```
@@ -182,6 +183,7 @@ def handle_items(self, items):
     @staticmethod
     def show_received_notification(items, players, locations):
         DialogHelper.create_notification(
-            'Received Items',
-            '\n'.join(
-                [f'{item} from {player} ({location})' for item, player, location in zip(items, players, locations)])).show_dialog()
+            S4APLocalizationUtils.create_from_string('Received Items'),
+            S4APLocalizationUtils.create_from_string(
+                '\n'.join(
+                    [f'{item} from {player} ({location})' for item, player, location in zip(items, players, locations)]))).show_dialog()
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)